pandas/core/generic.py

```diff
@@ -138,6 +138,9 @@ def _init_mgr(self, mgr, axes=None, dtype=None, copy=False):
 
     @property
     def _constructor(self):
+        """Used when a manipulation result has the same dimesions as the
+        original.
```

Comment:
I had actually just copied this docstring from [`internals.rst`](http://pandas.pydata.org/pandas-docs/stable/internals.html#override-constructor-properties).  Should I also the phrasing there too?